addons/account/data/account_reports_data.xml

```diff
@@ -13,12 +13,10 @@
                 <record id="generic_tax_report_column_net" model="account.report.column">
                     <field name="name">NET</field>
                     <field name="expression_label">net</field>
-                    <field name="figure_type">monetary_without_symbol</field>
```

Comment:
You'll need a migration script to convert everything using 'monetary_without_symbol' to 'monetary'